assets/bingo/lang/pl_pl.json

```diff
@@ -0,0 +1,411 @@
+	"bingo.lobby.tutorial.goals.20_no_bingo.heading": "20 Bez Bingo",
+	"bingo.lobby.tutorial.goals.20_no_bingo": "Zdobądź 20 przedmiotów tak, aby nie zdobyć żadnego Bingo.\n\nPlanując, pomyśl jakie przedmioty omijać, aby osiągnąć cel. Uważaj na przedmioty które można podnieść przypadkiem, takie jak strzały ze szkieletów.",
+	"bingo.lobby.tutorial.goals.multiple_bingo.heading": "Multi-Bingo",
+	"bingo.lobby.tutorial.goals.multiple_bingo": "Do zdobycia tego celu, zdobądź kilka Bingo jednocześnie.\n\nAby to zrobić, zbierz po 4 przedmioty w przecinających się kolumnach, rzędach lub na przekątnych i na koniec na ich przecięciu. Istnieją Bingo podwójne, potrójne i poczwórne.",
+
+	"bingo.lobby.tutorial.game_info.sign.line1": "",
+	"bingo.lobby.tutorial.game_info.sign.line2": "Informacje",
+	"bingo.lobby.tutorial.game_info.sign.line3": "o Grze",
+	"bingo.lobby.tutorial.game_info.sign.line4": "",
+
+	"bingo.lobby.tutorial.game_info.introduction.heading": "Informacje o Grze",
+	"bingo.lobby.tutorial.game_info.introduction": "Poniższa książka zawiera ogólne informacje o grze. Jest też kilka zmian konfiguracyjnych generatora świata, które są tu również opisane.",
+	"bingo.lobby.tutorial.game_info.general.heading": "Ogólne",
+	"bingo.lobby.tutorial.game_info.general": "Wszystkie zdobyte przedmioty muszą być dokładnie takie jak na karcie. Liczą się wszystkie przedmioty o tym samym wyglądzie (np. zaklęta książka nie wymaga konkretnego zaklęcia, ale czerwone łóżko już wymaga konkretnego koloru).",
+	"bingo.lobby.tutorial.game_info.world_gen.heading": "Zmiany Generatora Świata",
+	"bingo.lobby.tutorial.game_info.world_gen": "Minecraft Bingo ma pewne modyfikacje generatora świata.",
+	"bingo.lobby.tutorial.game_info.world_gen.tiny_biomes.heading": "Małe Biomy: ",
+	"bingo.lobby.tutorial.game_info.world_gen.tiny_biomes": "Biomy zostały zmniejszone dla ułatwienia eksploracji.\nZe względu na pewne ograniczenia, możesz zauważyć też inne różnice w ich układzie.",
+	"bingo.lobby.tutorial.game_info.world_gen.more_dungeons.heading": "Więcej Lochów: ",
+	"bingo.lobby.tutorial.game_info.world_gen.more_dungeons": "Aby uczynić lochy rozsądnymi do odnalezienia w szybkiej grze Bingo, ich liczba została zwiększona o współczynnik 12,5, dzięki czemu łatwo jest je znaleźć w każdej rundzie.",
+	"bingo.lobby.tutorial.game_info.world_gen.caves.heading": "Zmiany Jaskiń: ",
+	"bingo.lobby.tutorial.game_info.world_gen.caves": "Częstotliwość występowania jaskiń została nieznacznie zwiększona\n(z domyślnej 1/7 do 1/6).\n\nPonadto, biomy oceaniczne nie generują zalanych jaskiń na tej mapie.",
+	"bingo.lobby.tutorial.game_info.world_gen.no_mineshafts.heading": "Brak Kopalń: ",
+	"bingo.lobby.tutorial.game_info.world_gen.no_mineshafts": "Opuszczone kopalnie okazały się zbyt potężne, ponieważ dają łatwy dostęp do wielu skrzyń. Dlatego też w Bingo zostały one wyłączone.",
+	"bingo.lobby.tutorial.game_info.world_gen.ites.heading": "Brak -it'ów/-yt;ów: ",
```

Comment:
Maybe it would make sense to translate it as "no stone variants" as the ending is not the same across all of them. Might be easier to read like that?
I am also worried there might be other blocks which might end similarly that could cause confusion

Reply:
Yeah, that's true, will change it

Reply:
```suggestion
	"bingo.lobby.tutorial.game_info.world_gen.ites.heading": "Brak wariantów kamienia: ",
```